automatic/spotify/update.ps1

```diff
@@ -0,0 +1,43 @@
+Import-Module AU
+
+# We only use this for checking for Spotify versions, nothing else
+$releasesUrl = 'http://www.filehorse.com/download-spotify/'
+$downloadUrl = 'https://download.spotify.com/SpotifyFullSetup.exe'
+
+function global:au_BeforeUpdate {
+  # we need to verify that the downloaded file is the correct version
+  Get-RemoteFiles -Purge
+  $file = Get-ChildItem "$PSScriptRoot\tools" -Filter "*.exe" | select -First 1 | % { Get-Item $_.FullName }
```

Comment:
My mind just went mostly blank on this line, was the only thing I could remember when I updated the PR.
I'll change it to use `Get-Item` instead though when I merge it.